docs/source/conf.py

```diff
@@ -54,6 +54,8 @@
     "notfound.extension",
     "sphinxcontrib.jquery",
     "sphinx.ext.intersphinx",
+    "sphinx_last_updated_by_git",
```

Comment:
![image](https://github.com/kedro-org/kedro/assets/316517/b908ff52-fe37-4b5c-995a-e25cee42774a)

Better than nothing. Not very visible, but I understand that, to place this information wherever we want, we'd need https://github.com/jdillard/sphinx-gitstamp + a bit of templating magic

Reply:
I'm relatively happy with this level of visibility because I don't feel like it adds too much information as it stands. For example https://kedro--3547.org.readthedocs.build/en/3547/nodes_and_pipelines/nodes.html says it was updated in late November 2023 but we know that page is sadly in need of improvement. All we did then was fix a link or something.

What I'm actively looking into now (and I'm checking with the tech writer community because nothing is available for Sphinx) is a way to reflect confidence in a page (traffic light type indicator) so that readers know not only when the page was last touched but whether the team think it stands up or no. We used to do this at Improbable but it was a manual process...want to see who else does it and how.

Reply:
Maybe we could have two levels: "normal" (no news, good news) and "yellow light" (needs attention), so we could flag certain pages with page-wide metadata https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata. I think the equivalent in MyST would be https://myst-parser.readthedocs.io/en/latest/configuration.html#frontmatter-local-configuration ?

Essentially adding

```
---
needs_attention: true
---
```

to the top of the pages we think need attention, and create some Python logic that sees that and adds a banner.

Reply:
Sounds about right, let me think some more about levels because there may be different flavours: 

* This is content that needs some attention
* This is content that is factually OK but missing sections
* This content is probably outdated (uses and older version of a tool, not tested with latest). Buyer beware!
* This content needs the Kedro team to look at it (maybe a flag we add as needed but don't display)

etc. Let me do an audit of some pages and see. We don't want to get too complex but may as well make all the flag additions in one go.

Reply:
I've been through the tagging that Wikipedia use: https://en.wikipedia.org/wiki/Wikipedia:Template_index/Cleanup

From looking at them I suggest we have:

* Needs specific cleanup (link to GitHub issue required) needed to meet Kedro's high docs standards. The inclusion of a linked GitHub issue clarifies to the reader where the page is problematic, so they can use it with their best judgement.
* Potentially needs revision (no GH issue required) e.g. because of reliance on external libraries which may have changed. Warns reader that there may be problems, directs them to Slack/GH to let us know if they hit them.

We could also have something on each page to say "Did you find this useful" with a link to Slack or GitHub to tell us if not, but I think we've tried this level of docs feedback before (if not here, then I've done elsewhere) and you don't really get any feedback that we don't already know.